pandunia-lekse.csv

```diff
@@ -2165,7 +2165,7 @@ pluton||||Pluto (dwarf planet)||Plutón||||||||||||||Pluto (kääpiöplaneetta)|
 plutonium|mate|Pu||plutonium||plutonio|||||||||||||plutonio|plutonium|pluton|
 podia|||tib:བོད (bod, phØ)|Tibet||Tíbet|||||||||||||Tibeto|Tiibet|Tybet|
 poke|||eng:poker, zho:扑克 (pūkè), jpn:ポーカー (pōkā), kor:포커 (pokeo)|poker||póquer|||||||||||||pokero|pokeri|poker|
-pol|||ell:πολλοί (polloí), eng:fra:deu:poly-, spa:por:poli-, rus:поли- (poli-)|many (much)||muy|||||||||||||multe|paljon (monta)|wiele|
+pol (poli)|||ell:πολλοί (polloí), eng:fra:deu:poly-, spa:por:poli-, rus:поли- (poli-)|many (much)||muy|||||||||||||multe|paljon (monta)|wiele|
```

Comment:
Use the word with -i only.

Reply:
Do you think we should have "koy" and "pan" always be "koyi" and "pani" as well, then?